include/open62541/server.h

```diff
@@ -92,7 +92,7 @@ struct UA_ServerConfig {
      * ^^^^^^^^^^^^^^^^^^
      * The description must be internally consistent. The ApplicationUri set in
      * the ApplicationDescription must match the URI set in the server
-     * certificate. */
+     * certificate. The applicationType is not just descriptive, it changes the actual functionality of the server. */
```

Comment:
Explicitly mention `applicationType != UA_APPLICATIONTYPE_DISCOVERYSERVER`.

Reply:
/**
     * Server Description
     * ^^^^^^^^^^^^^^^^^^
     * The description must be internally consistent. The ApplicationUri set in
     * the ApplicationDescription must match the URI set in the server
     * certificate.
     * The applicationType is not just descriptive, it changes the actual
     * functionality of the server. The RegisterServer service is available only
     * if the server is a DiscoveryServer and the applicationType is set to the
     * appropriate value.*/
     
     like this?

Reply:
perfect